### DISCUSSION TITLE
Engine State: UNKNOWN: 3f3d0000 error in email

### DISCUSSION THREAD

**Discussion by biggen1684** [jgyates/genmon#1164] [Questions] [answered]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open issues](https://github.com/jgyates/genmon/issues),[closed issues](https://github.com/jgyates/genmon/issues?q=is%3Aissue+is%3Aclosed),[open discussion](https://github.com/jgyates/genmon/discussions), [closed discussion](https://github.com/jgyates/genmon/discussions/categories/questions?discussions_q=is%3Aclosed) - which may answer my question or that I could contribute my information to.
- [X] I have read the [Known Issues Wiki Page](https://github.com/jgyates/genmon/wiki/Appendix-D-Known-Issues) and my question / issue is not listed.
- [X] I have provided the version of the software and the operating system in my post.
- [X] If I have an issue with genmon not communicating with the controller, I have read and followed the [serial troubleshooting guide](https://github.com/jgyates/genmon/wiki/3.6---Serial-Troubleshooting).
- [X] I have included the generator make and model in my submission.

### Version

1.19.03

### Have you submitted your log files from the About page? Note that outbound email must be working for logs to be submitted.

I have submitted logs

### What is/are your question(s)?

Got this email today from my generator.  First I've ever seen of it.  According to the Wiki, I see the evolution controllers throw errors from time to time.  I don't see this particular error mentioned anywhere.  I also don't see where one can disable this via the "Ignore Unknown values" in advanced page.  I actually don't even see that option anywhere unless I have just overlooked it.

I have submitted logs and registers and I also have automated feedback enabled.

**Comment by jgyates** [comment, marked as answer]
This is covered in the Know Issues page:

https://github.com/jgyates/genmon/wiki/Appendix-D-Known-Issues#11-evolution-20-artifact

This is not an alarm but an anomaly of the Evolution 2.0 controllers, possibly related to the encryption that is unique top that controller. Let me know if you have any other questions on this.

**Reply by biggen1684** [reply]
Thanks for the quick response and I thought as much.  The wiki makes mention of enabling "Ignore Unknown Values".  I don't see this option anywhere.

**Reply by jgyates** [reply]
This setting is on the Advanced Settings page. To access the advanced settings page, in the web interface, click on the icon in the upper right once for live register view and a second time for the advanced settings page.
